release/libraries.txt

```diff
@@ -0,0 +1,53 @@
+Apache Commons (Apache-2.0)
+DesktopSupport (GPL-3.0-or-later)
+JAXB (CDDL-1.1 or GPL-2.0-with-classpath-exception)
+url-protocol-handler (GPL-3.0-or-later)
+Jackson (Apache-2.0)
+PDFBox (Apache-2.0)
+mALIGNa (MIT)
+DictZip (GPL-2.0-with-classpath-exception)
+trie4j (Apache-2.0)
+DSL4j (GPL-3.0-or-later)
+StarDict4j (GPL-3.0-or-later)
+juniversalchardet (LGPL-2.1-or-later or GPL-2.0)
+VLDocking Framework (LGPL-2.1-or-later)
+HTMLParser (LGPL-2.1-or-later)
+GNUDiff4j (GPL-1.0-or-later)
+lib-mnemonics (GPL-2.0)
+Jmyspell (LGPL-2.1-or-later)
+LanguageTool (LGPL-2.1-or-later)
+Lucene (Apache-2.0)
+Hunspell (LGPL-2.1-or-later or MPL-1.1 or GPL-2.0-or-later)
+JGit (EPL-2.0)
+JSch by Mwiede (BSD-3-Clause)
+Bouncy Castle (MIT-Modern-Variant)
+jUnixSocket (Apache-2.0)
+SVNKit (TMate)
+SuperTMXMerge (Apache-2.0)
+Jasypt (Apache-2.0)
+Groovy (Apache-2.0)
+rsyntaxtextarea (BSD-3-Clause)
+rstaui (BSD-3-Clause)
+languagesupport (BSD-3-Clause)
+languagetool (LGPL-2.1-or-later)
+autocomplete (BSD-3-Clause)
+slf4j (MIT)
+
+Transient dependencies:
+XMLRPC-client (Apache-2.0)
+ANT-ANTLR (Apache-2.0)
+ANTLR (BSD-3-Clause)
+Trilead-ssh2 (BSD-3-Clause)
+Caffeine (Apache-2.0)
+guava (Apache-2.0)
+protobuf-java (BSD-3-Clause)
+hamcrest (BSD-3-Clause)
+jsonic (Apache-2.0)
+diffutils (Apache-2.0)
+qdox (Apache-2.0)
+jzlib (BSD-3-Clause)
+jwordsplitter (Apache-2.0)
+opennlp (Apache-2.0)
+Morfologik FSA (BSD-3-Clause)
+OpenRegex (LGPL-2.1-or-later)
+SQLJet (GPL-2.0-only)
```

Comment:
GPL-2.0-only is incompatible with OmegaT's GPL-3.0-or-later. This is another issue.